src/components/Customer/PPMBooking/DatesAndLocation/DatesAndLocation.test.jsx

```diff
@@ -0,0 +1,326 @@
+import React from 'react';
+import { render, waitFor, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import DatesAndLocation from './DatesAndLocation';
+
+const defaultProps = {
+  onSubmit: jest.fn(),
+  onBack: jest.fn(),
+  serviceMember: {
+    id: '123',
+    residentialAddress: {
+      postalCode: '90210',
+    },
+  },
+  destinationDutyStation: {
+    address: {
+      postalCode: '94611',
+    },
+  },
+  postalCodeValidator: jest.fn(),
+};
+
+const mtoShipmentProps = {
+  onSubmit: jest.fn(),
+  onBack: jest.fn(),
+  serviceMember: {
+    id: '123',
+    residentialAddress: {
+      postalCode: '90210',
+    },
+  },
+  destinationDutyStation: {
+    address: {
+      postalCode: '94611',
+    },
+  },
+  postalCodeValidator: jest.fn(),
```

Comment:
Might be worth replacing with:
```suggestion
  ...defaultProps,
```